screen-capture-api/basic-screen-capture/index.css

```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0 25px;
+}
+
+#main-app {
+  display: flex;
+  gap: 5%;
+  min-width: 980px;
+}
+
+video,
+#demo {
+  flex: 1;
+}
+
+video,
+#demo>p,
```

Comment:
FYI, prettier flags all of these as

```css
#demo > p,
```

Reply:
That's weird. I originally wrote them as `#demo > p` and then ran Prettier on the code locally, and it changed them to this. And then the commit lint run didn't flag them.